crates/languages/src/json.rs

```diff
@@ -156,7 +157,8 @@ impl LspAdapter for JsonLspAdapter {
         if should_install_language_server {
             self.node
                 .npm_install_packages(&container_dir, &[(package_name, latest_version.as_str())])
-                .await?;
+                .await
+                .log_err();
```

Comment:
On windows, this will fail, the node log:
```txt
230 info run core-js@3.37.1 postinstall node_modules/core-js node -e "try{require('./postinstall')}catch(e){}"
231 info run core-js@3.37.1 postinstall { code: undefined, signal: undefined }
232 verbose stack TypeError [ERR_INVALID_ARG_TYPE]: The "file" argument must be of type string. Received undefined
232 verbose stack     at new NodeError (node:internal/errors:399:5)
232 verbose stack     at validateString (node:internal/validators:163:11)
232 verbose stack     at normalizeSpawnArguments (node:child_process:545:3)
232 verbose stack     at spawn (node:child_process:750:13)
232 verbose stack     at C:\Users\36477\AppData\Local\Zed\node\node-v18.15.0-win-x64\node_modules\npm\node_modules\@npmcli\promise-spawn\lib\index.js:18:12
232 verbose stack     at new Promise (<anonymous>)
232 verbose stack     at promiseSpawn (C:\Users\36477\AppData\Local\Zed\node\node-v18.15.0-win-x64\node_modules\npm\node_modules\@npmcli\promise-spawn\lib\index.js:17:13)
232 verbose stack     at spawnWithShell (C:\Users\36477\AppData\Local\Zed\node\node-v18.15.0-win-x64\node_modules\npm\node_modules\@npmcli\promise-spawn\lib\index.js:123:10)
232 verbose stack     at promiseSpawn (C:\Users\36477\AppData\Local\Zed\node\node-v18.15.0-win-x64\node_modules\npm\node_modules\@npmcli\promise-spawn\lib\index.js:12:12)
232 verbose stack     at runScriptPkg (C:\Users\36477\AppData\Local\Zed\node\node-v18.15.0-win-x64\node_modules\npm\node_modules\@npmcli\run-script\lib\run-script-pkg.js:79:13)
233 verbose cwd C:\Users\36477\AppData\Local\Zed\languages\json-language-server
234 verbose Windows_NT 10.0.22631
235 verbose node v18.15.0
236 verbose npm  v9.5.0
237 error code ERR_INVALID_ARG_TYPE
238 error The "file" argument must be of type string. Received undefined
239 verbose exit 1

```